pkg/exchange/bybit/stream.go

```diff
@@ -32,7 +32,8 @@ type Stream struct {
 	key, secret string
 	types.StandardStream
 
-	bookEventCallbacks []func(e BookEvent)
+	bookEventCallbacks   []func(e BookEvent)
+	walletEventCallbacks []func(e []*WalletEvent)
```

Comment:
use slice of value?

Reply:
oh it's a very large struct, ok let's use pointer LOL

Reply:
Yeah~ that's why I chose him in the end